modules/virt-autoupdate-custom-bootsource.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-autoupdate-custom-bootsource_{context}"]
+= Enabling automatic updates on custom boot sources
+
+{VirtProductName} automatically updates pre-defined boot sources by default. You must manually enable automatic imports and updates on any custom boot sources.
+
+.Procedure
+
+. Use the following command to enable automatic updates on a custom boot source:
++
+[source,terminal]
+----
+$ oc apply -n openshift-cnv -f add-custom-boot-sources.yaml
+----
+
++
+.Example in CentOS 7
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      name: centos7-image-cron
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: "true" <1>
+    spec:
+      schedule: "0 */12 * * *" <2>
+      template:
+        spec:
+          source:
+            registry: <3>
+              url: docker://quay.io/containerdisks/centos:7-2009
+          storage:
+            resources:
+              requests:
+                storage: 10Gi
+      managedDataSource: centos7 <4>
+      retentionPolicy: "None" <5>
+----
+<1> This annotation is required for storage classes with link:https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode[`volumeBindingMode`] set to `WaitForFirstConsumer`.
+<2> Schedule for the job specified in link:https://en.wikipedia.org/wiki/Cron[cron format].
+<3> Use to create a Data Volume from an registry source.
+<4> For the custom image to be detected as an available boot source, the name of the image's `managedDataSource` must match the name of the template's `DataSource`, which is found under `spec.dataVolumeTemplates.spec.sourceRef.name`.
+<5> Use `All` to retain `DataVolumes` and `DataSources` when `DataImportCron` is deleted. Use `None` to delete `DataVolumes` and `DataSources` when `DataImportCron` is deleted.
+
+[NOTE]
```

Comment:
maybe add a reference to this note in <3> ?

Reply:
Can do